entry.go

```diff
@@ -245,7 +245,13 @@ func (entry *Entry) fireHooks() {
 	defer entry.Logger.mu.Unlock()
 	err := entry.Logger.Hooks.Fire(entry.Level, entry)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to fire hook: %v\n", err)
+		if merr, ok := err.(multiErr); ok {
+			for _, e := range merr {
```

Comment:
Why not do formatting in `multiErr.Error()`?

Reply:
You mean move the `"Failed to fire hook: "` bit into multiErr's `Error()` method?

Reply:
No, that should probably stay here. I meant the ranging. I guess I could argue either case, so not sure.

Reply:
Oh right, I see what you mean. It's a good point.

This, and your other comments, have me really thinking about the best way to handle these multiple errors.